cube/src/main/java/org/jboss/arquillian/ce/cube/CEEnvironmentProcessor.java

```diff
@@ -142,18 +144,24 @@ private void registerRoutes(CECubeConfiguration configuration, OpenShiftClient c
     }
 
     private void processTemplate(TestClass tc, OpenShiftAdapter client, CECubeConfiguration configuration)
-        throws DeploymentException {
+            throws DeploymentException {
         final StringResolver resolver = Strings.createStringResolver(configuration.getProperties());
         final Template template = ReflectionUtils.findAnnotation(tc.getJavaClass(), Template.class);
         final String templateURL = readTemplateUrl(template, configuration, resolver);
         final List<? extends OpenShiftResource> resources;
+
+        if (templateURL == null || templateURL.length() == 0) {
```

Comment:
@spolti OK, makes sense.

Any way to determine 100% which tests require template and which don't?
(so we definitely fail on the one's that do require it)